diff --git a/ios/EXConstants.podspec b/ios/EXConstants.podspec
index fc4608aec9d7ac1e884cb6c57aaf0886f9ff6bbe..3d88e3a7e60164c62d278b4ccc0701efbbc273ea 100644
--- a/ios/EXConstants.podspec
+++ b/ios/EXConstants.podspec
@@ -37,7 +37,7 @@ Pod::Spec.new do |s|
   env_vars = ENV['PROJECT_ROOT'] ? "PROJECT_ROOT=#{ENV['PROJECT_ROOT']} " : ""
   script_phase = {
     :name => 'Generate app.config for prebuilt Constants.manifest',
-    :script => "bash -l -c \"#{env_vars}$PODS_TARGET_SRCROOT/../scripts/get-app-config-ios.sh\"",
+    :script => "SCRIPT=\"$PODS_TARGET_SRCROOT/../scripts/get-app-config-ios.sh\"; bash -l \"$SCRIPT\"",
     :execution_position => :before_compile
   }
   # :always_out_of_date is only available in CocoaPods 1.13.0 and later
