diff --git a/ios/EXAV/EXAV.m b/ios/EXAV/EXAV.m
index 7153d286adb56461bfecf916e164c932a17b5d56..7fb9276c61e90ebdb1aa41ceadc78166d0d01f20 100644
--- a/ios/EXAV/EXAV.m
+++ b/ios/EXAV/EXAV.m
@@ -326,6 +326,8 @@ EX_EXPORT_MODULE(ExponentAV);
   if (_allowsAudioRecording) {
     // Bluetooth input is only available when recording is allowed
     requiredAudioCategoryOptions = requiredAudioCategoryOptions | AVAudioSessionCategoryOptionAllowBluetooth;
+    // Route audio to speaker (not earpiece) in PlayAndRecord mode
+    requiredAudioCategoryOptions = requiredAudioCategoryOptions | AVAudioSessionCategoryOptionDefaultToSpeaker;
   }
   
   return [_kernelAudioSessionManagerDelegate setCategory:requiredAudioCategory withOptions:requiredAudioCategoryOptions forModule:self];
